test/elixir_console_web/features/console_test.exs

```diff
@@ -96,4 +97,13 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor gets Elixir and Erlang version", %{session: session} do
```

Comment:
While this is ok, I don't know if we want to add too many feature tests as these are slow. In particular, I'm not very concerned about introducing a regression here but this is questionable. Thoughts? cc/ @grzuy

Reply:
Yup, generally agree.

I think in this particular case, this a functionality that can be left out of the feature test coverage without losing a lot given the implementation is quite simple and straightforward, meaning there's low probability of introducing a bug, and if still buggy, it would be something can be easily detected visually by just rendering the page.

@matiaswilner Thanks anyway for trying to have coverage for this, better to err on the side of coverage than non.